src/install/pre_install.sh

```diff
@@ -43,8 +61,8 @@ then
 fi
 sudo usermod -aG docker "$FACTUSER"
 
-sudo -EH pip3 install --upgrade pip
-sudo -EH pip3 install --upgrade docker-compose
+sudo -EH pip3 install -U pip
```

Comment:
I would stay with `--upgrade` since it improves readability and it is used on all the other occurrences.

Reply:
There should be no other occurrences left, since installation was moved to python, but can do anyway.